examples/tutorial/step10.go

```diff
@@ -38,15 +38,17 @@ func mySplitter(cur *packet.Packet, ctx flow.UserContext) uint {
 	return cur.L3ACLPort(localL3Rules)
 }
 
-func myHandler(curV []*packet.Packet, num uint, ctx flow.UserContext) {
-	for i := uint(0); i < num; i++ {
-		cur := curV[i]
-		cur.EncapsulateHead(common.EtherLen, common.IPv4MinLen)
-		cur.ParseL3()
-		cur.GetIPv4NoCheck().SrcAddr = packet.BytesToIPv4(111, 22, 3, 0)
-		cur.GetIPv4NoCheck().DstAddr = packet.BytesToIPv4(3, 22, 111, 0)
-		cur.GetIPv4NoCheck().VersionIhl = 0x45
-		cur.GetIPv4NoCheck().NextProtoID = 0x04
+func myHandler(curV []*packet.Packet, mask *[32]bool, ctx flow.UserContext) {
```

Comment:
Replace 32 with some constant?
E. g.
const vecSize = 32
...
func myHandler(....mask *[vecSize]bool...
...
for .. i < vecSize